### DISCUSSION TITLE
From the ‘events’ menu, it is no longer possible to search for events in the past

### DISCUSSION THREAD

**Discussion by LottoGam** [kewisch/gdata-provider#786] [Support]
Hi, I am a problem.
From the ‘events’ menu, it is no longer possible to search for events in the past (All Events), but only: Events today; Events in the next 7/14/31 days... etc.

How can I also search (as was possible in previous versions) in the past?
My installed version of TB is 128.1.0esr, while Provider for Google Calendar is 128.0.0.

Thank you for your reply

**Comment by kewisch** [comment]
This is an issue in Thunderbird, please see https://bugzilla.mozilla.org/show_bug.cgi?id=1855900 for example.
